contrib/buildahimage/Containerfile

```diff
@@ -1,40 +1,59 @@
-# git/Dockerfile
+# [stable|testing|upstream]/Containerfile
+#
+# Build a Buildah container image from the latest version
+# of Fedora.
+#
+# FLAVOUR defaults to stable if unset
+#
+# FLAVOUR=stable    aquires a stable version of Buildah
+#                   from the Fedoras Updates System.
+# FLAVOUR=testing   aquires a testing version of Buildah
+#                   from the Fedoras Updates System.
+# FLAVOUR=upstream  aquires a testing version of Buildah
+#                   from the Fedora Copr Buildsystem.
+#                   https://github.com/containers/buildah
+#
+# https://bodhi.fedoraproject.org/updates/?search=buildah
 #
-# Build a Buildah container image from the latest
-# upstream version of Buildah on GitHub.
-# https://github.com/containers/buildah
 # This image can be used to create a secured container
 # that runs safely with privileges within the container.
-# The containers created by this image also come with a
-# Buildah development environment in /root/buildah.
 #
+
 FROM registry.fedoraproject.org/fedora:latest
+ARG FLAVOUR=stable
```

Comment:
Lol, the *other* spelling of "flavor" is going to drive my cide-spell-checker nuts :rofl: No worries, I can roll with it.

Reply:
Oh whoops! Sorry, I used English not American! Sorry sorry. I might be giving away what country I grew up in, eh!

I'll 'fix' it, since the majority here probably uses the American version. Minimize the number of people befuddled by the language!

Reply:
(meaning: please don't change it for my sake)

Reply:
No, I think it should be changed. There is value in using a consistent dialect of the language. Predictability and all. It is very in line with my intentions with this entire pull request. Simplify maintenance, lower cognitive load, etc.

See nalind's comment about my use of shell syntax where more simple containerfile syntax could be used in your first review topic for an example of what happens. Context switching and code switching have a cognitive cost.